### PR TITLE
fixed price change epoch bug

### PR DESCRIPTION
**Pull request by 0xModene**
Price Change epoch now set for the data range selected on the chart, where previously it was change since start

### CODE REVIEW

**Comment by 0xModene** [comment]
These changes _should_ be identical for every MarketData except the homepage one, which is minimally different, just doesn't need to account for changing price epoch

File: src/components/SimplePriceChart/SimplePriceChart.tsx [hunk 1]
```diff
@@ -69,19 +69,19 @@ const MarketDataChart: React.FC<SimplePriceChartProps> = ({
...
   }
   const handleWeeklyButton = () => {
     setDurationSelector(1)
-    setIsDaily(false)
+    setChartRange(7)
   }
   const handleMonthlyButton = () => {
     setDurationSelector(2)
-    setIsDaily(false)
+    setChartRange(30)
```
Comment (controtie) [inline src/components/SimplePriceChart/SimplePriceChart.tsx:84]: small knit: using an enum for these values might improve readibility

```
enum PriceChartRangeOption {
  DAILY_PRICE_RANGE = 1,
  WEEKLY_PRICE_RANGE = 7,
  MONTHLY_PRICE_RANGE = 30,
}

setChartRange(PriceChartRangeOption.DAILY_PRICE_RANGE)
```

**Review by controtie — APPROVED** [review]
Awesome work 🎉

File: src/components/SimplePriceChart/SimplePriceChart.tsx (hunk 1 above)
Comment (0xModene) [reply src/components/SimplePriceChart/SimplePriceChart.tsx:84]: That's a great point, easy change to make as well.

**Pull request merged**